test/testdata/httpd-deployment.yaml

```diff
@@ -1,14 +1,10 @@
 ---
-kind: PersistentVolumeClaim
 apiVersion: v1
+kind: ConfigMap
 metadata:
-  name: www-claim
-spec:
-  accessModes:
-    - ReadWriteOnce
-  resources:
-    requests:
-      storage: 1k
+  name: www-content
+data:
+  index.html:
```

Comment:
We can have this like following in case checked using `curl <route_url>` and validating the correct content. 
```
data:
  index.html: "<html>Hello CRC</html>"
```